src/pages/admin/reports/components/amount-edit-cell/AmountEditCell.module.scss

```
@@ -0,0 +1,71 @@
+    flex-direction: column;
+    align-items: flex-start;
+    justify-content: center;
+    height: 72px;
+    overflow: visible;
+    position: relative;
+    width: 128px;
+}
+
+.amount-edit-input {
+    @include type.subtitle-2-medium;
+    width: 128px;
```

Stylelint (17.12.0)

[error] Expected empty line before declaration (declaration-empty-line-before)

(declaration-empty-line-before)